core/classification.py

```diff
@@ -40,7 +40,12 @@ def kmeans_clustering_train(original_test_data, train_data, clf):
             clf.fit(clustered_train_data.drop('actual', 1), y)
 
             models[i] = clf
-    return kmeans_clustering_test(original_test_data, models, estimator, testing=True)
+    split=dict()
+    split['type']='double'
```

Comment:
What does the 'double' mean in this context? Please rename the type to something more clear. And use constants like i use `CLASSIFICATION` when setting the types.